platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/maps/widgets/CompassViewTest.java

```diff
@@ -40,19 +38,21 @@ public void testDefault() {
   }
 
   @Test
+  @Ignore
   public void testVisible() {
     validateTestSetup();
-
-    onView(withId(R.id.mapView)).perform(new MoveCameraAction(mapboxMap,
-        CameraUpdateFactory.newCameraPosition(
+    invoke(mapboxMap, new MapboxMapAction.OnInvokeActionListener() {
+      @Override
+      public void onInvokeAction(UiController uiController, MapboxMap mapboxMap) {
+        mapboxMap.moveCamera(CameraUpdateFactory.newCameraPosition(
           new CameraPosition.Builder()
             .bearing(45)
             .zoom(1)
             .target(new LatLng())
             .build()
-        )
-      )
-    );
+        ));
+      }
+    });
     waitLoop();
```

Comment:
To keep consistency with the rest of the tests we could use `uiController.loopMainThreadForAtLeast(500);` instead of `waitLoop();`.